contrib/macdeploy/background.svg

```diff
@@ -1,34 +1,32 @@
-<?xml version="1.0" standalone="no"?>
-<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 20010904//EN"
- "http://www.w3.org/TR/2001/REC-SVG-20010904/DTD/svg10.dtd">
-<svg version="1.0" xmlns="http://www.w3.org/2000/svg" width="1000pt" height="640pt" viewBox="0 0 1000 640" preserveAspectRatio="xMidYMid meet">
+<?xml version="1.0" encoding="utf-8"?>
 	<!-- kate: space-indent off;
-	Copyright (c) 2015 The Bitcoin Core developers
+	Copyright (c) 2019 The Bitcoin Core developers
 	Distributed under the MIT software license, see the accompanying
 	file COPYING or http://www.opensource.org/licenses/mit-license.php.
 	-->
-	<style type="text/css"><![CDATA[
-		text {
-			font-family: "Tuffy";
-			font-size: 86px;
-			fill: gray;
-			text-anchor: middle;
-		}
-	]]></style>
-	<defs>
-		<linearGradient id="gradient" x1="0%" y1="0%" x2="100%" y2="100%">
-			<stop offset="0%" style="stop-color:rgb(239,239,239);stop-opacity:1" />
-			<stop offset="33%" style="stop-color:rgb(239,239,239);stop-opacity:1" />
-			<stop offset="80%" style="stop-color:rgb(205,205,205);stop-opacity:1" />
-			<stop offset="100%" style="stop-color:rgb(204,204,204);stop-opacity:1" />
-		</linearGradient>
-	</defs>
-	<rect width="1000" height="640" style="fill:url(#gradient);stroke-width:0" />
-	<g transform="translate(500,0) scale(0.9, 1)">
-		<text x="0" y="114">PACKAGE_NAME</text>
-	</g>
-	<g transform="translate(0.000000,640.000000) scale(0.100000,-0.100000)"
-	fill="#000000" stroke="none">
-		<path d="M4995 3705 c-24 -23 -25 -29 -25 -165 l0 -140 -306 0 -306 0 -29 -29 c-29 -29 -29 -31 -29 -141 0 -110 0 -112 29 -141 l29 -29 306 0 306 0 0 -140 c0 -136 1 -142 25 -165 16 -17 35 -25 57 -25 29 0 72 32 306 226 180 149 274 233 278 250 13 53 -2 70 -278 299 -235 194 -277 225 -306 225 -22 0 -41 -8 -57 -25z" fixlter="url(#glow)"/>
-	</g>
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 500 320" style="enable-background:new 0 0 500 320;" xml:space="preserve">
+<style type="text/css">
+	.st0{fill:url(#SVGID_1_);}
+	.st1{fill:#808080;}
+	.st2{font-family:'Tuffy-Bold';}
+	.st3{font-size:12px;}
+	.st4{font-size:44px;}
+	.st5{font-size:10px;}
+</style>
+<linearGradient id="SVGID_1_" gradientUnits="userSpaceOnUse" x1="-1245.668" y1="163.2625" x2="-1245.168" y2="163.2625" gradientTransform="matrix(1000 0 0 640 1245668 -104328)">
+	<stop  offset="0" style="stop-color:#EFEFEF"/>
+	<stop  offset="0.33" style="stop-color:#EFEFEF"/>
+	<stop  offset="0.8" style="stop-color:#CDCDCD"/>
+	<stop  offset="1" style="stop-color:#CCCCCC"/>
```

Comment:
Are these actually being changed?

Reply:
I kept the same values but technically the gradient is different.

Reply:
The goal is to keep this configuration simple. You may notice that in all of the transformations - I have intentionally made them whole numbers. It is easier to read and maintain but basically creates the same effect. Future maintainers may not have the same tools I used so I am trying to consider future maintainability in this commit.

Reply:
NOTE: I thought I was done - I added the copyright notice but left out the MIT license notice.